From bc12001cb93b29799bc7cbbf963e4b793c008032 Mon Sep 17 00:00:00 2001
From: Fabian Vogt <fabian@ritter-vogt.de>
Date: Wed, 20 Mar 2024 09:38:27 +0100
Subject: [PATCH] Remove some features with questionable security

- Don't allow installing third-party themes from the store
- Don't recommend to clear the kwallet password
---
 src/ui/Advanced.qml | 1 -
 src/ui/main.qml     | 7 -------
 2 files changed, 8 deletions(-)

diff --git a/src/ui/Advanced.qml b/src/ui/Advanced.qml
index 8be5d81..3d05eea 100644
--- a/src/ui/Advanced.qml
+++ b/src/ui/Advanced.qml
@@ -36,7 +36,6 @@ Kirigami.Page {
                     // to appear when the user checks the checkbox, not all the
                     // time when the checkbox is checked.
                     if (checked && kcm.KDEWalletAvailable()) {
-                        autologinMessage.visible = true;
                     }
                 }
             }
diff --git a/src/ui/main.qml b/src/ui/main.qml
index c30dd1f..e8b3bdd 100644
--- a/src/ui/main.qml
+++ b/src/ui/main.qml
@@ -34,13 +34,6 @@ KCM.GridViewKCM {
             text: i18nc("@action:button", "Install From File…")
             icon.name: "document-import"
             onTriggered: themeDialog.open()
-        },
-        NewStuff.Action {
-            text: i18nc("@action:button as in, \"get new SDDM themes\"", "Get New…")
-            configFile: "sddmtheme.knsrc"
-            onEntryEvent: function(entry, event) {
-                kcm.themesModel.populate();
-            }
         }
     ]
     header: Kirigami.InlineMessage {
-- 
2.43.2

